assemblies/cdf-samples/src/main/resources/plugin-samples/pentaho-cdf/20-samples/blueprint/index.properties

```diff
@@ -1,12 +1,13 @@
-# Copyright 2006 - 2017 Hitachi Vantara.  All rights reserved.
-# This software was developed by Hitachi Vantara and is provided under the terms
-# of the Mozilla Public License, Version 1.1, or any later version. You may not use
+# Copyright 2002 - 2017 Webdetails, a Hitachi Vantara company. All rights reserved.
```

Comment:
@plagoa  2002?

Reply:
Wow, don't know how that ended up like that. Thanks for the heads up.

Reply:
Because the original file that you moved had that?